customization-base/src/main/java/com/azure/autorest/customization/implementation/ls/EclipseLanguageClient.java

```diff
@@ -71,33 +65,8 @@ public EclipseLanguageClient(String workspaceDir) {
 
     public EclipseLanguageClient(String pathToLanguageServerPlugin, String workspaceDir) {
         this.workspaceDir = new File(workspaceDir).toURI();
-        try {
-            serverSocket = new ServerSocket(0);
-            int port = serverSocket.getLocalPort();
-            Thread thread = new Thread(() -> {
-                try {
-                    clientSocket.set(serverSocket.accept());
-                } catch (IOException e) {
-                    throw new RuntimeException(e);
-                }
-            });
```

Comment:
I am not sure why we remove this?

Reply:
The PR changed it so we don't use sockets anymore to connect to the language server. Now it's using the standard in and out from the JVM process running the language server to communicate, effectively the same that Autorest does with autorest.java!